### DISCUSSION TITLE
About M1 Developers and Extensions

### DISCUSSION THREAD

**Discussion by addison74** [OpenMage/magento-lts#1884] [General]
In the last month I talked to more M1 and OM administrators, developers. A relatively small part has been oriented towards M2, another larger part continues to provide support for M1 and OM and there is another part about 10% that wants to implement other solutions. 

Nothing new so far. The problem that those who manage M1 and OM have is that there are either extensions but no support from developers or there are none at all. The new variants of PHP may cause issues with those extensions. If the stores work very well and no new facilities are needed then you can continue with this as much as possible. From what I see OM is still a solid project, has a small community, new versions periodically are released . But it will be a real issue with extensions that will no longer be supported. 

Whoever wants to continue with M1 will have to put the store in the arms of a person interested in managing it. There are fewer and fewer who do. But M2 is not a solution either as long as you need high-performance, configured systems, more expensive extensions. Although many believed that OM did not manage to survive after 1 and a half years the project goes ahead. I hope that we will be able to solve all the small problems reported at least out of the desire not to lose our knowledge and not to let this project die suddenly.

**Comment by henryhayes** [comment]
I'll address this upside down.

> From what I see OM is still a solid project, has a small community, new versions periodically are released . But it will be a real issue with extensions that will no longer be supported.

I'm relatively new to this project as I have a very many M1 projects and clients and am happy to join the cause. But, since we're now observing semantic versioning, the rule is clear, that we cannot break backwards compatability without a major version change; and we're sticking to OM 1.9.4.x - so theoretically, won't be an issue.

> The problem that those who manage M1 and OM have is that there are either extensions but no support from developers or there are none at all. The new variants of PHP may cause issues with those extensions.

If they're listed in GitHub and we can establish a tag, something like #om-update-required (or something similar), we as developers can work through them proposing patches.

**Comment by fballiano** [comment]
I think this is kinda what we discussed in https://github.com/OpenMage/magento-lts/discussions/1806
The projects I'm managing are not working on PHP8 (cause of the unmaintained extensions) so... I still don't know what I'll do, rector didn't seem to work although I'll try again sometimes in the future.
